### PR TITLE
Create newsletter_21_Feb2021.md draft

### LINKED ISSUE #1718: Collect news item for February 2021 newsletter
### Summary

* The next newsletter will be sent on 17 January 2021
* Any news item can be posted below as a comment

### What needs to be done?

- [ ] Collect all the items and share in the news letter

### Who can help?

* Anyone from the community who has something to share 👍 

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

Fixes #1718 

### List of changes proposed in this PR (pull-request)

* Add draft
* Add images
* Ask @LauraCarter to verify her entry
* Include Zenodo for a new release

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by malvikasharan** [comment]
@LauraCarter can you please approve the section mentioning your work?

File: communications/newsletters/newsletter_21_Feb2021.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+> When designing, developing, trialling and deploying emerging technologies, we aspire to work in the open wherever we can, sharing our plans and experience, working collaboratively with other organisations and reusing good practice.
+> Use of open, robust and reproducible approaches and standards, as illustrated, for example, by *The Turing Way* community materials.
+
+## Acknowledgements and celebrations
+
+### Contributor in focus: Laura Carter
+
+![Head shot of Laura Carter](images/2021-02-laura-carter-3.png)
+
+Laura Carter is a PhD candidate in the Human Rights Centre at the University of Essex, UK, researching the human rights implications of the use of data-driven technologies in the UK public sector, focusing on gender stereotyping and gender discrimination. She brings her experience from working as a human rights researcher for almost a decade, specialising mostly in human rights, sexual orientation and gender identity. She has been contributing to the development of *The Turing Way* Guide for Ethical Research and has co-authored chapters including the introduction to ethical research and activism in data science.
```
Comment (LauraCarter) [inline communications/newsletters/newsletter_21_Feb2021.md:81]:
```suggestion
Laura Carter is a PhD candidate in the Human Rights Centre at the University of Essex, UK. She is researching the human rights implications of the use of data-driven technologies in the UK public sector, focusing on gender stereotyping and gender discrimination. She brings her experience from working as a human rights researcher for almost a decade, specialising in gender, sexual orientation and gender identity. She has been contributing to the development of *The Turing Way* Guide for Ethical Research and has co-authored chapters including the introduction to ethical research and activism in data science.
```

**Review by LauraCarter — APPROVED** [review]
Looks good to me!

File: communications/newsletters/newsletter_21_Feb2021.md (hunk 1 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_21_Feb2021.md:81]: Thank you @LauraCarter <3

**Comment by BatoolMM** [comment]
Sorry, I corrected a tiny detail about the workshop and the talk because they are totally separate events (unrelated to each other), and thanks for the mention.

**Comment by malvikasharan** [comment]
@KirstieJane's comment: Newsletter is good to go (with maybe a request to change to just onboarding sessions rather than community catch up).

**Comment by malvikasharan** [comment]
Oh @BatoolMM I just saw your edits. I have mentioned your events separately but I definitely got the dates wrong in the newsletter. My apology! I will fix that here.

**Pull request merged**